fuel-core/src/cli/run/p2p.rs

```diff
@@ -147,17 +155,26 @@ impl From<P2pArgs> for anyhow::Result<P2PConfig> {
             }
         };
 
+        // Reserved nodes do not count against the configured peer input/output limits.
```

Comment:
We should test this for sentries, which have a reserved node but can also connect to other non-reserved peers.